staging/src/k8s.io/legacy-cloud-providers/vsphere/vclib/utils.go

```diff
@@ -203,31 +203,83 @@ func VerifyVolumePathsForVMDevices(vmDevices object.VirtualDeviceList, volPaths
 
 // isvCenterDeprecated takes vCenter version and vCenter API version as input and return true if vCenter is deprecated
 func isvCenterDeprecated(vCenterVersion string, vCenerAPIVersion string) (bool, error) {
```

Comment:
nit: can you please fix typo `vCenerAPIVersion`?

Reply:
fixed.